pennylane/collections/apply.py

```diff
@@ -37,12 +37,13 @@ def apply(func, qnode_collection):
     As we are using the ``'torch'`` interface, we now apply ``torch.sum``
     to the QNodeCollection:
 
-    >>> cost = qml.apply(torch.sum, qnodes)
+    >>> cost = qml.collections.apply(torch.sum, qnodes)
```

Comment:
That was a bug...`qml.apply` is now another function.

Reply:
nice catch 😬